### DISCUSSION TITLE
Mark checkbox condition.

### DISCUSSION THREAD

**Discussion by vaskoZGU** [Budibase/budibase#12037] [Help] [answered]
Hello,

In my form I have Option picker field and checkbox. 
How can I mark (set true) the checkbox based on selected option/s only?
I know how to create condition for enable/disable the checkbox, but I don't know how to make it for changing the value (true/false).

Thank you.

**Comment by andz-bb** [comment, marked as answer]
hey @vaskoZGU 

You can do this by adding an **Update Field Value** On change action to your Option picker component. Select the checkbox field that you would like to update and use a binding for the value that returns true or false depending on the field value (i.e. the selected option in the picker).

Here's the JavaScript binding I'm using which will return **true** if the selected city is in the list or **false** if it's not:

```js
return ['Belfast', 'Cork', 'Derry'].includes($("Field Value"))
```

Here's a screenshot showing how I set this up for a simple example:

<img width="1512" alt="update field value action" src="https://github.com/Budibase/budibase/assets/110921612/3083cc4b-b51d-480e-9e97-e7c673a218fe">


Now when I select **Derry** in my options picker, the value of my checkbox is set to _true_ but when I select **Liverpool** the value changes to _false_.

![checkbox value based on selected option](https://github.com/Budibase/budibase/assets/110921612/ee0c0fbf-d8ef-4eb3-9792-810ff2d087b5)

Hope this helps 😊

**Reply by vaskoZGU** [reply]
Hey Andrew,

Thank you very much for your reply. I was looking in the wrong direction.
By any chance do you know how to invert selection in JavaScript i.e. mark the checkbox if the city selected is not in 'Belfast', 'Cork', 'Derry'?
